apps/system/js/camera_trigger.js

```diff
@@ -0,0 +1,28 @@
+/* globals BaseModule */
+
+'use strict';
+
+(function(exports) {
+  var CameraTrigger = function() {};
+
+  BaseModule.create(CameraTrigger, {
+    name: 'CameraTrigger',
+    DEBUG: true,
+    TRACE: false,
+
+    _start: function nsm_start() {
+      window.addEventListener('holdcamera', this.triggerCamera.bind(this));
```

Comment:
better to be consistent:
CameraTrigger.EVENTS = ['holdcamera'];
rename triggerCamera to _handle_holdcamera
then BaseModule will do all favor for you